### DISCUSSION TITLE
How created the icons?

### DISCUSSION THREAD

**Discussion by buhtz** [themesberg/flowbite#612] [Q&A]
Do I get it right that [Evelyne Krall](https://twitter.com/EvelyneKrall) is the creator of all that icons?

Or did you just collected the icons from third party sources?

How can I get in contact with the creator?

**Comment by zoltanszogyenyi** [comment]
Hey @buhtz,

Please reach out to us via the contact form as this is sensitive information.

All of the icons are the intellectual property of Bergside which is the company behind Flowbite.

Thanks!
